docs/reference/announcements-release-notes/880/880-announcements.md

```diff
@@ -1,65 +1,97 @@
 ---
-id: announcements-880
-title: "8.8 Announcements"
-description: "Important changes and updates for the Camunda 8.8 release including deprecation & removal notices."
+id: 880-announcements
+title: "Announcements"
+description: "Supported environment changes and breaking changes or deprecations for the Camunda 8.8 release."
+toc_max_heading_level: 2
 ---
 
-import DeployDiagramImg from '../img/deploy-diagram-modal.png';
+import DeployDiagramImg from '../../img/deploy-diagram-modal.png';
+import AnnouncementsGrid from '../../react-components/\_announcements-card.js';
+import { AnnouncementsCards } from '../../react-components/\_announcements-card-data';
 
-Important changes and updates for the Camunda 8.8 release are summarized below.
+Supported environment changes and breaking changes or deprecations for the Camunda 8.8 release are summarized below.
+
+[Release notes](/reference/announcements-release-notes/880/880-release-notes.md) now contain all updates for new features and enhancements. Click any of the features below to be re-routed to the release notes.
```

Comment:
I don't think we have to describe the change, just the new status-quo.

```suggestion
Please check the [Release notes](/reference/announcements-release-notes/880/880-release-notes.md) to find a summary about all new features and enhancements.
```

Reply:
Adjusting in incoming commit 👍